tests/unit/composition/parse-standalone.test.ts

```
@@ -0,0 +1,509 @@
+
+  for (const [atomic_number_str, amount] of Object.entries(
+    atomic_composition,
+  )) {
+    const atomic_number = Number(atomic_number_str)
+    const symbol = atomic_number_to_element_symbol(atomic_number)
+
+    if (!symbol) {
+      throw new Error(`Invalid atomic number: ${atomic_number}`)
+    }
+
+    if (typeof amount === `number` && amount > 0) {
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)